R/g.part5.analyseRest.R

```diff
@@ -0,0 +1,191 @@
+g.part5.analyseRest = function(sibreport = NULL, dsummary = NULL,
+                                ds_names = NULL, fi = NULL, di = NULL,
+                                time = NULL, tz = NULL, possible_nap_dur = 0,
+                               possible_nap_edge_acc = Inf) {
+  # Only consider sib episodes with minimum duration
+  if (length(grep(pattern = "mean_acc_1min", x = colnames(sibreport))) > 0) {
+    sibreport$acc_edge = pmax(sibreport$mean_acc_1min_before, sibreport$mean_acc_1min_after)
+  } else {
+    sibreport$acc_edge = 0
+  }
+  longboutsi = which((sibreport$type == "sib" &
+                        sibreport$duration >= possible_nap_dur[1] &
+                        sibreport$acc_edge <= possible_nap_edge_acc) |
+                       (sibreport$type != "sib" & sibreport$duration >= 1))
+  # for qc purposes:
+  dsummary[di,fi] = length(longboutsi)
+  ds_names[fi] = "sibreport_n_items"
+  fi = fi + 1
+  if (length(longboutsi) > 0) {
+    sibreport = sibreport[longboutsi,]
+    srep_tmp = sibreport[which(sibreport$start > min(time) &
+                                 sibreport$end < max(time)),]
```

Comment:
I was not getting the expected output when testing this, and it is because time here is introduced as ISO6801, but `sibreport$start` and `sibreport$end` are stored as POSIX objects, so the function was not able to find any `srep_tmp`. 
I fixed this locally by introducing these lines at the begining of this function:
```
# transform time to POSIX
  if (is.ISO8601(time[1])) {
    time = iso8601chartime2POSIX(time, tz = tz)
  }
```
@vincentvanhees Should I commit this change or do you want to investigate why do you get the expected output and I don't?